concent_api/middleman/middleman_server.py

```diff
@@ -137,7 +161,7 @@ def _close_middleman(self) -> None:
                 self._is_signing_service_connection_active = False
 
     async def _respond_to_user(self, data: bytes, writer: asyncio.StreamWriter) -> None:  # pylint: disable=no-self-use
-        writer.write(data)
+        writer.write(append_frame_separator(escape_encode_raw_message(data)))
```

Comment:
It should be done by the async clone of `middleman.protocol.stream.send_over_stream` function.

Reply:
done